### ISSUE TITLE
Is there no validation set in the VIL-100 dataset?

### ISSUE THREAD

**Issue by anhuidinglieqiaodaima** [Sephirex-X/ADNet#15]
VIL-100 does not have a validation set. How can we ensure that there is no overfitting or underfitting during training?
